platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitInteractiveIntegrationTest.groovy

```diff
@@ -18,20 +18,30 @@ package org.gradle.buildinit.plugins
 
 import org.gradle.buildinit.plugins.fixtures.ScriptDslFixture
 import org.gradle.buildinit.plugins.internal.modifiers.BuildInitDsl
-import org.gradle.internal.jvm.Jvm
 import org.gradle.test.fixtures.ConcurrentTestUtil
 import org.gradle.util.internal.TextUtil
 import spock.lang.Issue
 
 class BuildInitInteractiveIntegrationTest extends AbstractInitIntegrationSpec {
 
-    String projectTypePrompt = "Select type of project to generate:"
-    String dslPrompt = "Select build script DSL:"
-    String incubatingPrompt = "Generate build using new APIs and behavior (some features may change in the next minor release)?"
-    String basicType = "1: basic"
-    String projectNamePrompt = "Project name (default: some-thing)"
-    String convertMavenBuildPrompt = "Found a Maven build. Generate a Gradle build from this?"
-    List<String> languageSelectionOptions = ["Select implementation language:", "1: C++", "2: Groovy", "3: Java", "4: Kotlin", "5: Scala", "6: Swift"]
+    def buildTypePrompt = "Select type of build to generate:"
+    def dslPrompt = "Select build script DSL:"
+    def incubatingPrompt = "Generate build using new APIs and behavior (some features may change in the next minor release)?"
```

Comment:
I think this question is very cumbersome for the users as well. Without the context, it begs the question of why would new APIs change in the minor release?

I think it would be much better to just say "Use incubating Gradle APIs?". For those who know, the question is short and clear. For those who don't know, the word "incubating" gives a hint, and the first Google result for "Gradle incubating" already gives the full answer. It also is better connected to the CLI option `--incubating` that corresponds to this question.

@lkasso, could you give your input here as well?

Reply:
I never understood what it meant and had to ask @big-guy when I first used it.
I agree with @alllex, `Use incubating Gradle APIs?` or similar is a much better choice.

Reply:
For sure. I didn't change that prompt in this PR (that's a test refactoring). It's on the list for later.